### PR TITLE
docs(bpmn-workflows): allow multiple start events

### PR DESCRIPTION
**Pull request by saig0**
## Description

* update documentation that multiple start events are allowed

## Related issues

implemented by #3317 

## Pull Request Checklist

- [x] All commit messages match our [commit message guidelines](https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md#commit-message-guidelines)
- [ ] The submitting code follows our [code style](https://github.com/zeebe-io/zeebe/wiki/Code-Style)
- [ ] If submitting code, please run `mvn clean install -DskipTests` locally before committing

### CODE REVIEW

File: docs/src/bpmn-workflows/message-events/message-events.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ At the moment, messages can be published only externally by using one of the Zee
 
 ## Message Start Events
 
-A workflow can have one or more message start events. Each of the message events must have a unique message name. If the workflow has at least one message start event then it must not have a none start event. Other types of start events are allowed.
+A workflow can have one or more message start events (beside other types of start events). Each of the message events must have a unique message name.
```
Comment (npepinpe) [inline docs/src/bpmn-workflows/message-events/message-events.md]: Super minor but I think `beside` is always written `besides`

**Review by npepinpe — APPROVED** [review]
:+1:

**Comment by saig0** [comment]
bors retry

**Pull request merged**